pandas/tests/extension/test_integer.py

```diff
@@ -234,3 +234,14 @@ class TestParsing(base.BaseParsingTests):
 
 class Test2DCompat(base.Dim2CompatTests):
     pass
+
+
+def test_concat_to_empty_ea():
+    """`concat` to an empty EA should maintain type EA information."""
+    df_empty = pd.DataFrame({"a": pd.array([], dtype=pd.Int64Dtype())})
+
+    df_new = pd.DataFrame({"a": pd.array([1, 2, 3], dtype=pd.Int64Dtype())})
+
+    result = pd.concat([df_empty, df_new])
+
+    assert result["a"].dtype == df_empty["a"].dtype == df_new["a"].dtype
```

Comment:
Could you build an `expected` DataFrame and use `tm.assert_frame_equal`? It will compare the dtypes too.